include/cuco/detail/open_addressing/open_addressing_ref_impl.cuh

```diff
@@ -1054,6 +1044,35 @@ class open_addressing_ref_impl {
     }
   }
 
+  /**
+   * @brief Attempts to insert an element into a slot.
+   *
+   * @note Dispatches the correct implementation depending on the container
+   * type and presence of other operator mixins.
+   *
+   * @note `stable` here means that the payload will only be updated once from the sentinel value to
+   * the payload value
```

Comment:
I'm confused by this note.

Maybe "from the sentinel value to the **desired** value"?